documentation/src/main/asciidoc/integrationguide/chapters/services/extras/extend/EventPublishingServiceImpl.java

```diff
@@ -19,10 +19,10 @@ public void injectServices(ServiceRegistryImplementor serviceRegistry) {
         this.serviceRegistry = serviceRegistry;
     }
 
-    public void configure(Map configurationValues) {
-        this.jmsConnectionFactoryName = configurationValues
+    public void configure(Map<String, Object> configurationValues) {
+        this.jmsConnectionFactoryName = (String) configurationValues
             .get( JMS_CONNECTION_FACTORY_NAME_SETTING );
-        this.destinationName = configurationValues
+        this.destinationName = (String) configurationValues
             .get( JMS_DESTINATION_NAME_SETTING );
```

Comment:
I added generics usage as much as possible in this PR for using `Map` alone seems sloppy